### ISSUE TITLE
Rendering issues

### ISSUE THREAD

**Issue by arashsa** [dunovank/jupyter-themes#117]
I'm experiencing som rendering issues on latest Chrome. When I scroll up and down the page, what has been outside of view, seems like it gets re-rendered. I'm no expert in frontend or css, so have no idea what may cause this. Regular Jupyter notebooks without themes does not do this.

**Comment by dunovank** [comment]
Yeah I've had that experience as well. Super annoying. I've found a workaround by changing some of the settings in `chrome://flags` (Just copy and paste `chrome://flags` that into your url bar to open, then Ctrl-F to find the following options).

Scroll Prediction (switch to enabled)
Scroll Anchoring (switch to enabled)
Threaded scrolling (switch to disabled)

I'm actually not sure which one of these ended up fixing the issue but I haven't noticed any adverse side effects of making these changes so I just left them. Hopefully that works for you.

**Comment by arashsa** [comment]
It seems like disabling threaded scrolling did the trick.

**Comment by dunovank** [comment]
Cool. I'm going to leave this open so people can find the workaround until a fix is submitted.

**Comment by arashsa** [comment]
Yes, would be great if there was a fix, as I think having threaded scrolling disabled might reduce performance of Chrome.

**Comment by dunovank** [comment]
Any clue as to what would cause this?

**Comment by cjeiler** [comment]
This is great help, much appreciated. It could be that [this](https://github.com/jupyterlab/jupyterlab/issues/816) is related, not sure but figured I'd point it out.

**Comment by nikolayvoronchikhin** [comment]
@dunovank Threaded scrolling (switch to disabled) helped resolve this issue for Chrome. Is there a similar solution for Firefox, Safari, IE, and Edge?

**Comment by pheaktra1** [comment]
Can anyone help me please?
After installing jupyterthemes my jupyter server not responding.
How could i fix it? 
![11](https://user-images.githubusercontent.com/35774902/48671795-e5db0200-eb70-11e8-97f8-d3f5f5b7d9c6.PNG)

**Comment by przutto** [comment]
> @dunovank Threaded scrolling (switch to disabled) helped resolve this issue for Chrome. Is there a similar solution for Firefox, Safari, IE, and Edge?

for EDGE just like this
edge://flags

**Comment by souraOP** [comment]
Browsers like arc and Thorium doesn't have such Flags as of the latest version... hopefully everything is fixed by now
